### DISCUSSION TITLE
sourceURL:capacitor error

### DISCUSSION THREAD

**Discussion by m4rinos** [ionic-team/capacitor#3243] [Q&A]
Hi, I've been working on my first vue, ionic, capacitor ios app and so far is going well, but a few times in the process I've ran into a rather (for me) obscure error: 

`[error] - {"line":1,"column":3730,"sourceURL":"capacitor://localhost/js/chunk-07ff1c0d.66f9bf5e.js"}`. 

My situation is that I start an  audiocontext sound pattern, my sound plays in a loop, I stop it, start it a few times, no problem, then all of a sudden I cannot start my session anymore, every-time I click start, I get that error.

First I thought I just forgot  to run `npx cap sync`, that had solved similar issues in the past, but no luck. I've tried to completely rebuilt my app, changed the name even, tried in the xcode simulator as well as an actual phone, same issue. The web app is working fine.

Can someone clarify the error for me and perhaps hint me on how to debug such situations?

thanks!

**Comment by MarcStoecker** [comment]
Just found out together with a co-worker.:

Check if the device is in **energy save mode**.
